src/main/java/org/opentripplanner/framework/geometry/GeometryUtils.java

```diff
@@ -183,6 +187,34 @@ public static SplitLineString splitGeometryAtFraction(Geometry geometry, double
     return new SplitLineString(beginning, ending);
   }
 
+  /**
+   * Splits a line string into a number of partition with the same number of coordinates.
+   * <p>
+   * The last coordinate of partition n will also be the first coordinate of partition n+1 so all
+   * partitions together will represent the original line string without gaps.
+   *
+   * @param segmentSize How many coordinates should be in each partition.
```

Comment:
There is none. I probably changed my mind half-way through writing the Javadoc.

I now consistently use "segment": c34842cc309c10385ba9ba504c97ecbe348f7577

The method name still uses "partition" since "segmentize" probably isn't a word. Is that ok?